src/main/resources/database/1/patch_11.sql

```diff
@@ -56,7 +56,6 @@ BEGIN
 END;
 $BODY$;
 
-DROP VIEW IF EXISTS repbot_schema.user_reputation;
```

Comment:
Is this deletion intended?

Reply:
Yes it is. The current view has a depending view. I cant drop it directly. And since the view doesn't change the format I must not drop it.